xpano/constants.h

```diff
@@ -47,6 +47,9 @@ constexpr int kAboutBoxWidth = 70;
 constexpr int kAboutBoxHeight = 30;
 constexpr int kSidebarWidth = 25;
 
+constexpr int kBugReportBoxWidth= 70;
```

Comment:
You can make these values a bit smaller, so there isn't that much empty space around the text. If the ImGuiWindowFlags_AlwaysAutoResize works (from another comment), please remove both of these constants.